src/pages/base/wallets/components/WalletInfo.js

```diff
@@ -58,23 +59,42 @@ function WalletInfo(props) {
     },
     {
       title: "health rate",
-      bigValue: health_rate,
+      bigValue: (
+        <Row className="mb3">
+          <Col className="d-flex align-items-center">
+            <div>{health_rate}</div>
+          </Col>
+          <Col className="d-flex align-items-center">
+            {data.user_mode === 1 ? (
+              <Badge style={{ fontSize: "0.8rem" }} color="primary">
+                e-mode
+              </Badge>
+            ) : null}
+          </Col>
+        </Row>
```

Comment:
Replace this part with 
```suggestion
        <div className="d-flex align-items-center">
          <div className="flex-grow-1">{health_rate}</div>
          {data.user_mode === 1 ? (
            <Badge style={{ fontSize: "0.8rem" }} color="primary">
              e-mode
            </Badge>
          ) : null}
        </div>
```